vscode/content/en/docs/Extension Settings.md

```diff
@@ -98,11 +98,11 @@ Default: `True`
 
 #### puppet.installDirectory
 
-The fully qualified path to the Puppet install directory. This can be a PDK or Puppet Agent installation. For example: 'C:\Program Files\Puppet Labs\Puppet' or '/opt/puppetlabs/puppet'. If this is not set the extension will attempt to detect the installation directory
```

Comment:
Did you mean to remove this?

Reply:
That was automated so... yes?

Reply:
why tho? We still have these settings and we haven't changed the text for them?

Reply:
Ahhh found it.  The text changed from `description` to `markdownDescription`

Reply:
Updated and changed this to only update to 0.28.0.  The extension doesn't yet have changelog for 1.0.0